modules/hcp-configuring-oauth-console.adoc

```diff
@@ -24,7 +24,7 @@ Adding any identity provider in the OAuth configuration removes the default `kub
 
 [NOTE]
 ====
-When you configure identity providers, you must have at least one `NodePool` replica in your hosted cluster. However, the `htpasswd` and `request-header` identity providers do not need the `NodePool` replicas.
+When you configure identity providers, you must have at least one `NodePool` replica in your hosted cluster because, traffic for DNS resolution is sent through the worker nodes. However, the `htpasswd` and `request-header` identity providers do not need the `NodePool` replicas.
```

Comment:
```suggestion
When you configure identity providers, you must have at least one `NodePool` replica in your hosted cluster because traffic for DNS resolution is sent through the worker nodes. However, the `htpasswd` and `request-header` identity providers do not need the `NodePool` replicas.
```